scRNA-seq/05-clustering_markers_scRNA.Rmd

```diff
@@ -249,20 +249,25 @@ purrr::map(
 
 This returns a list of data frames, which isn't quite what we want.
 
-There is no built-in function that will give us just the first few _row names_, so we will have to define one, and `purrr` gives us a nice shorthand for doing that with `~` syntax.
-In this syntax we type a `~` followed by an expression (R code) that uses `.x` as a placeholder for a single element of the list.
-`purrr::map()` will then apply the expression to each element of the list, and return the results as a new list.
+There is no built-in function that will give us just the first few _row names_, so we will have to define one.
+As of version 4.1, R introduced a new approach to defining _anonymous functions_ - that is, functions you can quickly define "on-the-fly" without formally assigning them.
+They are handy when you need to do a very short task that requires a function, but it isn't really a function you need beyond this context.
+This new anonymous syntax looks like this: `\(x)...` (or for slightly longer code, use curly braces as in `\(x) {...}`).
+This would define an anonymous function that takes one argument, `x`, and `...` is where you would put the expression to calculate, aka the "body" of the function.
```

Comment:
I started by removing the repetition of `anonymous` here, but I moved into a bit more revision. I didn't think the "a.k.a." really added much, so I just removed that. (I also wasn't sure what "aka" was referring to, as I don't think there was really another name there!)
```suggestion
This defines a function that takes one argument, `x`, with `...` indicating where you would put the expression to calculate.
```